src/opera_disp_tms/__main__.py

```diff
@@ -25,7 +25,7 @@ def main():
     # NOTE: Will need to update to `entry_points(group='hyp3', name=args.process)` when updating to python 3.10
     # eps = entry_points()['hyp3']
     # (process_entry_point,) = {process for process in eps if process.name == args.process}
-    process_entry_point = entry_points(group='hyp3', name=args.process)
+    process_entry_point = list(entry_points(group='hyp3', name=args.process))[0]
```

Comment:
The comment on line 24 and commented implementation on line 27 works a little better here as it de-duplicates entrypoints and ensures you're only getting one.